src/config/env_config.go

```diff
@@ -0,0 +1,68 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/spf13/viper"
+)
+
+type EnvConfig struct {
+	TelegramID      []int64 `mapstructure:"TELEGRAM_ID"`
+	TelegramToken   string  `mapstructure:"TELEGRAM_TOKEN"`
+	EditWaitSeconds int     `mapstructure:"EDIT_WAIT_SECONDS"`
+}
+
+func (e *EnvConfig) HasTelegramID(id int64) bool {
+	for _, v := range e.TelegramID {
+		if v == id {
+			return true
+		}
+	}
+	return false
+}
+
+// LoadEnvConfig loads config from .env file, variables from environment take precedence if provided.
+func LoadEnvConfig(path string) (*EnvConfig, error) {
+	if !fileExists(path) {
+		return nil, fmt.Errorf("config file '%s' does not exist", path)
+	}
```

Comment:
Following #46, we want to proceed when the `.env` file isn't found (but print a warning just in case)

Reply:
there is a slight issue with this (a quirk of viper). it requires reading a config into viper to register the keys, only these keys can then be loaded from env. 

imo we should rename `env.example` to `.env` and keep the failure.

One other alternative would be to register the keys by encoding an empty instance of the `EnvConfig` we have, before reading from env, wdyt @m1guelpf ?

Reply:
@m1guelpf updated with a somewhat of a middle ground, introduced an `emptyConfig` string for the purpose of registering the keys. 

Test added for when no file is provided